app/Community/Services/ActivePlayersService.php

```diff
@@ -0,0 +1,118 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Community\Services;
+
+use Illuminate\Support\Carbon;
+use Illuminate\Support\Facades\Cache;
+
+class ActivePlayersService
+{
+    public function loadActivePlayers(?string $searchValue = null, bool $fetchAll = false): array
+    {
+        $allActivePlayers = Cache::remember(
+            'currently-active',
+            Carbon::now()->addMinutes(2),
+            function () {
+                return collect(getLatestRichPresenceUpdates())
+                    ->keyBy('User')
+                    ->map(function ($user) {
+                        $user['InGame'] = true;
+
+                        return $user;
+                    });
```

Comment:
This changes the data type stored in the cache key (by losing the `values()->toArray()`):
```
    fn () => collect(getLatestRichPresenceUpdates())
        ->keyBy('User')
        ->map(function ($user) {
            $user['InGame'] = true;

            return $user;
        })
        ->values()
        ->toArray()
```

Which leads to an error for the two minutes after switching the code over:
```
Call to a member function map() on array
```

Reply:
I think the best solution here is to just change the cache key and let the old one expire.

In latest, I've renamed it to "currently-active:20230921".